src/common/components/SocialShare.js

```diff
@@ -62,21 +62,164 @@ export const SocialShareContextProvider = props => {
   );
 };
 
+const useCopy = content => {
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => () => setCopied(false), []);
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(content);
+    setCopied(true);
+  };
+
+  return { copied, copyToClipboard };
+};
+
+const CopyEmbededCode = () => {
+  const isSmall = useMediaQuery(theme.breakpoints.down('sm'));
+  const { t } = useTranslation();
+  const {
+    socialShareProps: { embedUrl },
+  } = useContext(SocialShareContext);
+
+  const embedCode = useMemo(() => {
+    const iframe = document.createElement('iframe');
+    iframe.setAttribute('src', embedUrl);
+    iframe.setAttribute('title', t('share.embed_title'));
+    iframe.setAttribute('width', '100%');
+    iframe.setAttribute('height', '100%');
```

Comment:
issue: this is not valid HTML. height and width need to be integer values (in pixels).